.eslintrc.js

```diff
@@ -26,5 +26,27 @@ module.exports = {
 		// do not throw an error if imported dependencies are 
 		// declared in `package.json` or `gutenberg/package.json`
 		"import/no-extraneous-dependencies": ["error", {"packageDir": ['.', './gutenberg/']}] 
-	}
+	},
+	overrides: [
+		{
+			// Disable rules for tests as we do in Gutenberg.
+			// Reference: https://git.io/JSMAQ
+			files: [ 'src/**/test/**/*.js' ],
+			rules: {
+				'import/default': 'off',
+				'import/no-extraneous-dependencies': 'off',
+				'import/no-unresolved': 'off',
+				'import/named': 'off',
+				'@wordpress/data-no-store-string-literals': 'off',
+			},
```

Comment:
This rules override was taken from Gutenberg's ESLint configuration ([reference](https://github.com/WordPress/gutenberg/blob/43d950788629daaf588c80a2a56173124f112c60/.eslintrc.js#L178-L184)).